src/us/kbase/userprofile/UserProfileServer.java

```diff
@@ -243,11 +265,18 @@ public void updateUserProfile(SetUserProfileParams p, AuthToken authPart) throws
      * @param   usernames   instance of list of original type "username"
      * @return   parameter "users" of mapping from original type "username" to type {@link us.kbase.userprofile.GlobusUser GlobusUser}
      */
-    @JsonServerMethod(rpc = "UserProfile.lookup_globus_user")
-    public Map<String,GlobusUser> lookupGlobusUser(List<String> usernames, AuthToken authPart) throws Exception {
+    @JsonServerMethod(rpc = "UserProfile.lookup_globus_user", async=true)
+    public Map<String,GlobusUser> lookupGlobusUser(List<String> usernames, AuthToken authPart, RpcContext jsonRpcContext) throws Exception {
         Map<String,GlobusUser> returnVal = null;
         //BEGIN lookup_globus_user
-    	Map<String, UserDetail> data = AuthService.fetchUserDetail(usernames, authPart);
+
+        ConfigurableAuthService authService = new ConfigurableAuthService(
+                                                        new AuthConfig()
+                                                            .withKBaseAuthServerURL(authServiceUrl)
```

Comment:
You should add 'auth-service-url-allow-insecure' optional config parameter support like in https://github.com/kbase/narrative_method_store/pull/48/files .